{"name": "test_invalid_sign_in_get_request", "status": "passed", "steps": [{"name": "Fail sign in get request", "status": "passed", "start": 1648801589463, "stop": 1648801589659}], "start": 1648801589462, "stop": 1648801589660, "uuid": "3414b2d6-f4c3-43b3-8a89-c08687dfb01e", "historyId": "b76b91d68b162f208ab544833704df5e", "testCaseId": "c03d7c083b451bb970c52430f0e4a590", "fullName": "tests.api_testcases.login_test.login_test.TestUserLogin#test_invalid_sign_in_get_request", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.login_test"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "118790-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.login_test.login_test"}]}